strawberry/asgi/handlers/http_handler.py

```diff
@@ -35,13 +36,10 @@ async def handle(self, scope: Scope, receive: Receive, send: Send):
         request = Request(scope=scope, receive=receive)
         root_value = await self.get_root_value(request)
 
-        sub_response = Response(
-            content=None,
-            status_code=None,  # type: ignore
-            headers=None,
-            media_type=None,
-            background=None,
-        )
+        sub_response = Response()
+        if sub_response.status_code == 200:
+            # Set default status code to None
+            sub_response.status_code = None  # typing: ignore
```

Comment:
Hmm... Maybe just doing the same as I did for FastAPI? :thinking:

Reply:
@TimPansino have you seen this? 

@Kludex are you talking about this change? https://github.com/tiangolo/fastapi/pull/4483/files#diff-aef3dac481b68359f4edd6974fa3a047cfde595254a4567a560cebc9ccb0673fL465

Reply:
yes

Reply:
I like it, much simpler.